src/library_pthread.js

```diff
@@ -6,7 +6,6 @@
 var LibraryPThread = {
   $PThread__postset: 'if (!ENVIRONMENT_IS_PTHREAD) PThread.initMainThreadBlock(); else PThread.initWorker();',
   $PThread__deps: ['$PROCINFO', '_register_pthread_ptr',
-                   'emscripten_main_thread_process_queued_calls',
```

Comment:
Split out this change along with src/deps_info.json?  Seems unrelated but maybe I'm wrong?

Reply:
It only started failing in this branch because of the deps structure changing.